src/Components/UserList/index.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './style.module.css';
+import Status from '../Status';
+
+const testData = [
+  {
+    name: "Zach Gallaway",
+    businessName: "Food 4 U",
+    dateRegistered: "2020/07/07",
+    status: "active"
+  },
+  {
+    name: "Maxwell Countryman Skewes",
+    businessName: "Max's Country Meats",
+    dateRegistered: "2018/06/12",
+    status: "pending"
+  },
+  {
+    name: "Ian Scott",
+    businessName: "Scotts Tots",
+    dateRegistered: "2014/01/11",
+    status: "inactive"
+  },
+  {
+    name: "Jerry Goodman",
+    businessName: "Goodman Grocers",
+    dateRegistered: "2017/04/12",
+    status: "incomplete"
+  },
+  {
+    name: "Freddie Mercury",
+    businessName: "Chocolate Voices",
+    dateRegistered: "2017/03/28",
+    status: "suspended"
+  },
+  {
+    name: "Bill Burr",
+    businessName: "BB's BBQ",
+    dateRegistered: "2020/07/4",
+    status: "closed"
+  },
+]
+
+function UserList (props) {
+  const { data } = props;
+
+  return (
+      <table>
+        <tr>
+          <th>No.</th>
+          <th>Name</th>
+          <th>Business Name</th>
+          <th>Date Registered</th>
+          <th>Status</th>
+        </tr>
+
+        {/* Replace testData.map with line below for production
+        {data.map((entry, index) => {  */}
+        {testData.map((entry, index) => {
+          return(
+            <tr key={index}>
```

Comment:
React isn't too happy with us using the index: "Do not use Array index in keys". Will you please update this 😀

Reply:
Updated the key to use a UUID to resolve this issue.